app/models/change_note.rb

```diff
@@ -54,7 +53,7 @@ def update_type
   end
 
   def change_note_instance
-    edition.change_note || edition.create_change_note!(document: document)
+    edition.change_note || edition.create_change_note!(edition_id: edition.id)
```

Comment:
In fact, do we even need to pass `edition` here? Feels like that would be handled by `create_change_note!`.